### DISCUSSION TITLE
Why `bands` instead of channels?

### DISCUSSION THREAD

**Discussion by iansebryk** [python-pillow/Pillow#5341] [Q&A] [answered]
if i'm reading the docs correctly, the `bands` you refer to are image channels.  as `channel` is long- and well-established terminology, may i ask why you chose to use `band` instead?  and i appreciate this library has been around a while, but this is horribly confusing especially when using Pillow in a production pipeline...  :P

**Comment by radarhere** [comment, marked as answer]
`bands` has been in use since PIL, before it was forked to Pillow, so I don't know the historical answer.

To change this would require updating `Image.merge()`, `Image.getmodebandnames()`, `Image.getmodebands()`, `im.getbands()` and `ModeDescriptor`. Pillow values backwards compatibility, and this would disrupt people who have accepted the current terminology. So unless this topic receives a lot of attention, there isn't a strong motivation to change.
